README.md

```diff
@@ -252,3 +252,4 @@ Output | Description
 -------|------------
 `was_successful` | Whether or not the changes could be backported successfully to all targets. Either `true` or `false`.
 `was_successful_by_target` | Whether or not the changes could be backported successfully to all targets - broken down by target. Follows the pattern `{{label}}=true\|false`.
+`successful_prs` | Space separated list of the newly created PRs.
```

Comment:
I'd like to rename this to `created_pull_numbers` to allow further expansion of the number of outputs that may have some overlap with this one, e.g. an output that provides more data about the created pull requests.